scripts/apex/hello.apex

```diff
@@ -1,10 +1,38 @@
-// Use .apex files to store anonymous Apex.
-// You can execute anonymous Apex in VS Code by selecting the
-//     apex text and running the command:
-//     SFDX: Execute Anonymous Apex with Currently Selected Text
-// You can also execute the entire file by running the command:
-//     SFDX: Execute Anonymous Apex with Editor Contents
-
-string tempvar = 'Enter_your_name_here';
-System.debug('Hello World!');
-System.debug('My name is ' + tempvar);
\ No newline at end of file
+system.debug( StringGenerator.randomString( new List<String>{'aa','bb','cc'} ) );
+
+system.debug( StringGenerator.randomString( new List<String>{'aa','bb','cc'} ) );
```

Comment:
alway use right case- System.debug - as a standard of using debug statements add LoggingLevel for the debugs

Reply:
Noted